### DISCUSSION TITLE
Prioritising pull requests

### DISCUSSION THREAD

**Discussion by EnricoMi** [PyGithub/PyGithub#2321] [General]
Given the little time maintainers have to review and release pull requests, lets bring them to their attention here and vote on their priority.

Add links to your pull requests here and upvote your most wanted pull requests, or use emojis: 
![grafik](https://user-images.githubusercontent.com/44700269/193206158-446ab67f-93e2-4e31-a126-f2ac4c49f4a8.png)

Please do not vote using a comment.

Alternative to upvoting in this discussion, :+1: your favourite pull requests to get maintainers' attention: https://github.com/PyGithub/PyGithub/issues/2178#issuecomment-1263184406

**Comment by EnricoMi** [comment]
Throttle requests to mitigate `RateLimitExceededExceptions`
#2145 fixing #1989 (#1235, #2113, #2005, #1319, #1115, #810)

**Comment by EnricoMi** [comment]
Make `Requester.__createException` robust against missing message and body
#2159 fixing #1399 and #2158

**Comment by lelegard** [comment]
Add class Artifact
#2319 fixing #2313

**Comment by EnricoMi** [comment]
Let's stick to the :+1: vote of issues and pull requests, details: https://github.com/PyGithub/PyGithub/issues/2178#issuecomment-1263184406
